controllers/alphacontrollers/alert_controller.go

```diff
@@ -212,11 +216,17 @@ func (r *AlertReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl
 		log.V(1).Info("Alert was updated", "alert", jstr)
 	}
 
+	if flattenErr != nil {
+		log.Error(err, "Received an error while flattened Alert")
```

Comment:
```suggestion
		log.Error(flattenErr, "Received an error while flattened Alert")
```

Reply:
We could also move this upwards, and return the error even before we try to call Status.Update

Reply:
I prefer the status to be updated with nil integration-name